data/endlessos-dynamic.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE wallpapers SYSTEM "gnome-wp-list.dtd">
+<wallpapers>
+  <wallpaper deleted="false">
+    <name>Endless OS Default</name>
+    <filename>/usr/share/eos-media/desktop-background-C.jpg</filename>
+    <filename-dark>/usr/share/eos-media/desktop-background-dark-C.jpg</filename-dark>
```

Comment:
These two files will also be listed in the `endlessos-backgrounds.xml` file as separate wallpapers, so I think we'll get 3 entries in Settings: one for the combined day/night one, and one each for the individual files. This seems undesirable.

Reply:
The way they're presented, I might actually be happy with this; if someone prefers just the light or the dark version, they can explicitly choose to always use it _or_ to use the dynamic version. The dynamic one is distinct since it shows the two versions side-by-side.

That said, it looks like putting them in a sub-folder might prevent them from showing up standalone in the list?? I can test this.

(We want to improve this layout upstream to make it more clear between dynamic and standalone wallpapers, but there's some technical blocker last I checked)

Reply:
Can confirm, the individual wallpapers plus the dynamic one show up. I'm okay with this for the aforementioned reason, but I could understand wanting to not have the individual ones displayed, too.

![Screenshot from 2024-04-19 13-25-32](https://github.com/endlessm/eos-media/assets/611168/7238c7a0-a1c0-47ee-9040-6335f2921f8b) | ![Screenshot from 2024-04-19 13-24-18](https://github.com/endlessm/eos-media/assets/611168/6e45807d-8cad-4ac5-933d-2ab6d69cddbb)
--- | ---